src/shared/validation.ts

```diff
@@ -352,6 +352,7 @@ export const isSyncedState = z.strictObject({
           name: z.string(),
           level: z.number().int().min(0).max(9),
           prepared: z.boolean(), // or "yes" | "no" | "always" ?
+          concentrationRounds: z.number().int().min(0),
```

Comment:
I think this will not allow `Infinity`, but also see my comment above about JSON-encodability.